### DISCUSSION TITLE
Windows game bar (Windows + G or XBOX key on controller) isn't visible when streaming

### DISCUSSION THREAD

**Discussion by Gaeus** [LizardByte/community#146] [Sunshine] [answered]
Game bar will open, but the overlay isn't visible on the client device. It's the only overlay with whom this issue occur.

It's somewhat annoying, as the overlay is the easiest way to quit a bugged out fullscreen application when streaming on a phone.

**Comment by ReenigneArcher** [comment, marked as answer]
It's a known issue. #832
